src/CrowdedMod/Components/ShapeShifterPagingBehaviour.cs

```diff
@@ -19,6 +20,24 @@ public ShapeShifterPagingBehaviour(IntPtr ptr) : base(ptr)
     public IEnumerable<ShapeshifterPanel> Targets => shapeshifterMinigame.potentialVictims.ToArray();
 
     public override int MaxPageIndex => (Targets.Count() - 1) / MaxPerPage;
+    private TextMeshPro PageText = null!;
+
+    public override void Update()
+    {
+        base.Update();
+
+        if (!PageText)
+        {
+            PageText = Instantiate(HudManager.Instance.KillButton.cooldownTimerText, shapeshifterMinigame.transform);
```

Comment:
Any components should be instantiated in Awake or similar methods, unless you had bugs which happens in unity modding

Reply:
added it to the Start() method by making it virtual